Proposals/2

```diff
@@ -0,0 +1,103 @@
+---
+period: 7 
+title: Livestream Integration 
+author: Christian Casini ( @Niftgen )
+company_name:Niftgen Inc  
+category: dApp 
+focus_area: SocialFi, web 3 social, and NFTs 
```

Comment:
must be one of: `Banking`, `Defi`, `Dex`, `Gaming`, `Identity`, `Marketplace`, `Metaverse`, `NFT`, `Oracle`, `Storage`, `User Onboarding`, `IDE`, `Teal`, `Deployment`, `Libraries`, `Monitoring`, `Node`, `Education`

Reply:
@SudoWeezy Ok, the closest thing would be User Onboarding. I think SocialFi should be added as a category.

Reply:
New discussion was created here 
https://github.com/algorandfoundation/ARCs/pull/200